README.md

```diff
@@ -0,0 +1,80 @@
+# Project Setup
+
+## Prerequisites
+
+- Node >=20
+- Nvm
+- Install SubSquid CLI:
+
+```bash
+ npm i -g @subsquid/cli@latest
+```
+
+- Install Docker Desktop (<https://www.docker.com/products/docker-desktop/>)
+
+- Install PgAdmin 4 (<https://www.postgresql.org/ftp/pgadmin/pgadmin4/v7.8/macos/>)
+
+- Register on Dwellir to get a CHAIN_RPC_ENDPOINT URL (<https://dashboard.dwellir.com/register>) (It supports most chains)
+
```

Comment:
You can use the []() pattern for links as it looks a bit better to hide the long urls. (e.g. [Install PgAdmin 4](https://www.docker.com/products/docker-desktop/))

Not sure what the <> are doing in the link as well.

Reply:
I have to type it separately as it doesnt appear otherwise, the pattern is: "[" "]" "(" ")"